Whisky/Extensions/View+Extension.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  View+Extension.swift
+//  Whisky
+//
+//  This file is part of Whisky.
+//
+//  Whisky is free software: you can redistribute it and/or modify it under the terms
+//  of the GNU General Public License as published by the Free Software Foundation,
+//  either version 3 of the License, or (at your option) any later version.
+//
+//  Whisky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+//  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+//  See the GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License along with Whisky.
+//  If not, see https://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+
+extension View {
+    func fixedVerticalSize() -> some View {
```

Comment:
I'm not sure they're necessary as in `.fixedSize(horizontal: false, vertical: true)` is also a one-liner, so I don't see the need for a shortcut.

Reply:
It's a one-liner, but in my opinion, it's too verbose. You have to pass two arguments every time instead of just using the one that truly matters. However, if I'm not convincing enough, I can delete this extension.

Reply:
Done